docs/articles_en/openvino-workflow-generative/inference-with-genai/inference-with-genai-on-npu.rst

```diff
@@ -22,7 +22,7 @@ Install required dependencies:
 
          python3 -m venv npu-env
          npu-env/bin/activate
```

Comment:
don't we need 'source' in these commands?

Reply:
fixed